### PR TITLE
Implementing simple business card

### PR DESCRIPTION
**Pull request by Juliana-Dimeglio**
Implementation of exercise two, "Simple Business Card". I defined a Safe Area, made use of Circle avatar to create the image, changed the icon and the application title and added 3 texts below the image  
ejercicio2
IconoEjercicio2

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,113 +1,62 @@
...
+      title: 'Business Card',
+      home: Scaffold(
+        backgroundColor: Colors.blue.shade100,
+        body: SafeArea(
+          child: Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                CircleAvatar(
+                  backgroundImage: AssetImage("assets/images/avatar.jpeg"),
```
Comment (MarcosCGlobant) [inline lib/main.dart]: missed a trailing comma here

File: lib/main.dart [hunk 2]
```diff
@@ -1,113 +1,62 @@
...
+                  backgroundColor: Colors.red,
+                  radius: 100,
```
Comment (MarcosCGlobant) [inline lib/main.dart]: you could do a constants file an add this value in it

File: lib/main.dart [hunk 3]
```diff
@@ -1,113 +1,62 @@
...
+                ),
+                Text(
+                  'Juliana Dimeglio',
+                  style: TextStyle(
+                    fontSize: 45.0,
```
Comment (MarcosCGlobant) [inline lib/main.dart]: you can add this to the constants file

File: lib/main.dart [hunk 4]
```diff
@@ -1,113 +1,62 @@
...
+                    fontWeight: FontWeight.bold,
+                    foreground: Paint()..shader = linearGradient,
+                  ),
+                  textAlign: TextAlign.center,
+                ),
+                Text(
+                  'Flutter college trainee',
+                  style: TextStyle(
+                    fontSize: 25.0,
```
Comment (MarcosCGlobant) [inline lib/main.dart]: you can add this to the constants file

File: lib/main.dart [hunk 5]
```diff
@@ -1,113 +1,62 @@
...
+                  textAlign: TextAlign.center,
+                ),
+                Text(
+                  'juliana.dimeglio@globant.com',
+                  style: TextStyle(
+                    decoration: TextDecoration.underline,
+                    fontWeight: FontWeight.bold,
+                    decorationColor: Colors.indigo.shade900,
+                    decorationThickness: 1.5,
+                    fontSize: 20.0,
```
Comment (MarcosCGlobant) [inline lib/main.dart]: you can add this to the constants file

**Review by MarcosCGlobant — COMMENTED** [review]
Some trailing commas and could add a constants file

**Review by FranciscoGaraventa — APPROVED** [review]
After Harry comments, LGTM

File: lib/main.dart (hunk 5 above)
Comment (Juliana-Dimeglio) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 4 above)
Comment (Juliana-Dimeglio) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 3 above)
Comment (Juliana-Dimeglio) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 2 above)
Comment (Juliana-Dimeglio) [reply lib/main.dart]: Done

File: lib/main.dart (hunk 1 above)
Comment (Juliana-Dimeglio) [reply lib/main.dart]: Done

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

File: lib/main.dart [hunk 6]
```diff
@@ -1,113 +1,63 @@
 import 'package:flutter/material.dart';
+import 'utils/constants_file.dart';
```
Comment (MarcosCGlobant) [inline lib/main.dart:2]: relative import here please

File: lib/utils/constants_file.dart [hunk 7]
```diff
@@ -0,0 +1,6 @@
+abstract class Constants{
+  static const double radius = 100.0;
+  static const double nameFontSize = 45.0;
+  static const double textFontSize = 25.0;
+  static const double emailFontSize = 20.0;
+}
```
Comment (MarcosCGlobant) [inline lib/utils/constants_file.dart]: ctrl + alt + L here to fix the missing line, please

**Review by MarcosCGlobant — COMMENTED** [review]
2 minor comments

File: lib/utils/constants_file.dart (hunk 7 above)
Comment (Juliana-Dimeglio) [reply lib/utils/constants_file.dart]: Done

**Review by MarcosCGlobant — APPROVED** [review]

**Pull request merged**